dashboards/c-sharp/dashboard.json

```diff
@@ -0,0 +1,888 @@
+{
```

Comment:
I'm seeing a lot of entity GUIDs and Ids in this file, let's get those removed in favor of more general queries!